setup/gcp/installer/__init__.py

```diff
@@ -12,8 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-""" Forseti Security Installer Environment. """
-
+"""installer folder"""
```

Comment:
Not really ```folder```, but rather ```package```.  And please capitalize `Installer` and add a period at the end.

Reply:
Updated.